public/assets/cms/preview.js

```diff
@@ -8,12 +8,12 @@
+var DateFormatter;!function(){"use strict";var t,e,o,i,r,n;r=864e5,n=3600,t=function(t,e){return"string"==typeof t&&"string"==typeof e&&t.toLowerCase()===e.toLowerCase()},e=function(t,o,i){var r=i||"0",n=t.toString();return n.length<o?e(r+n,o):n},o=function(t){var e,i;for(t=t||{},e=1;e<arguments.length;e++)if(i=arguments[e])for(var r in i)i.hasOwnProperty(r)&&("object"==typeof i[r]?o(t[r],i[r]):t[r]=i[r]);return t},i={dateSettings:{days:["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"],daysShort:["Sun","Mon","Tue","Wed","Thu","Fri","Sat"],months:["January","February","March","April","May","June","July","August","September","October","November","December"],monthsShort:["Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],meridiem:["AM","PM"],ordinal:function(t){var e=t%10,o={1:"st",2:"nd",3:"rd"};return 1!==Math.floor(t%100/10)&&o[e]?o[e]:"th"}},separators:/[ \-+\/\.T:@]/g,validParts:/[dDjlNSwzWFmMntLoYyaABgGhHisueTIOPZcrU]/g,intParts:/[djwNzmnyYhHgGis]/g,tzParts:/\b(?:[PMCEA][SDP]T|(?:Pacific|Mountain|Central|Eastern|Atlantic) (?:Standard|Daylight|Prevailing) Time|(?:GMT|UTC)(?:[-+]\d{4})?)\b/g,tzClip:/[^-+\dA-Z]/g},DateFormatter=function(t){var e=this,r=o(i,t);e.dateSettings=r.dateSettings,e.separators=r.separators,e.validParts=r.validParts,e.intParts=r.intParts,e.tzParts=r.tzParts,e.tzClip=r.tzClip},DateFormatter.prototype={constructor:DateFormatter,parseDate:function(e,o){var i,r,n,s,a,p,h,u,l,c,f=this,y=!1,d=!1,g=f.dateSettings,v={date:null,year:null,month:null,day:null,hour:0,min:0,sec:0};if(!e)return void 0;if(e instanceof Date)return e;if("number"==typeof e)return new Date(e);if("U"===o)return n=parseInt(e),n?new Date(1e3*n):e;if("string"!=typeof e)return"";if(i=o.match(f.validParts),!i||0===i.length)throw new Error("Invalid date format definition.");for(r=e.replace(f.separators,"\x00").split("\x00"),n=0;n<r.length;n++)switch(s=r[n],a=parseInt(s),i[n]){case"y":case"Y":l=s.length,2===l?v.year=parseInt((70>a?"20":"19")+s):4===l&&(v.year=a),y=!0;break;case"m":case"n":case"M":case"F":isNaN(s)?(p=g.monthsShort.indexOf(s),p>-1&&(v.month=p+1),p=g.months.indexOf(s),p>-1&&(v.month=p+1)):a>=1&&12>=a&&(v.month=a),y=!0;break;case"d":case"j":a>=1&&31>=a&&(v.day=a),y=!0;break;case"g":case"h":h=i.indexOf("a")>-1?i.indexOf("a"):i.indexOf("A")>-1?i.indexOf("A"):-1,c=r[h
```

Comment:
__[jshint]__
* Expected a conditional expression and instead saw an assignment.
* Expected an assignment or function call and instead saw an expression.
* Confusing use of '!'.
* Missing '()' invoking a constructor.
* If a strict mode function is executed using function invocation, its 'this' value will be undefined.

<!-- Comment by [SideCI](https://sideci.com) -->